posix/subsystem/src/requests.cpp

```
@@ -698,6 +698,9 @@ async::result<void> serveRequests(std::shared_ptr<Process> self,
 			}else if(req->fs_type() == "devpts") {
 				co_await target.first->mount(target.second, pts::getFsRoot());
 			}else{
+				if(req->fs_type() != "ext2") {
+					std::cout << "posix: Trying to mount unsupported FS of type: " << req->fs_type() << std::endl;
+				}
 				assert(req->fs_type() == "ext2");
```

This should return an error.